### PR TITLE
some providers updates

### PR DESCRIPTION
**Pull request by jospoortvliet**
- fix link of unixcorn
- add three providers
- update one log

Signed-off-by: Jos Poortvliet <jospoortvliet@gmail.com>

### CODE REVIEW

**Comment by splitt3r** [comment]
Not 100% related but 
image misses two flag images.

File: assets/providers.json [hunk 1]
```diff
@@ -1404,5 +1404,47 @@
             "Wir bieten NextCloud als eigene Instanz oder Shared in unserer hochverfügbaren NetApp Infrastruktur in Österreich an. DSGVO konform und ISO 27001 zertifiziert."
```
Comment (splitt3r) [inline assets/providers.json]: NextCloud :disappointed:

Comment (MariusBluem) [reply assets/providers.json]: ..even if unrelated to this PR - I'll change this 🤔 Thank you for keeping your eyes open.

**Comment by jospoortvliet** [comment]
@splitt3r good catch, fixed the flags.

**Pull request merged**